icd/generated/mock_icd.cpp

```diff
@@ -31,7 +31,7 @@ using std::unordered_map;
 // Map device memory handle to any mapped allocations that we'll need to free on unmap
 static unordered_map<VkDeviceMemory, std::vector<void*>> mapped_memory_map;
 
-static VkPhysicalDevice physical_device = nullptr;
+static VkPhysicalDevice physical_device = (VkPhysicalDevice)CreateDispObjHandle();
```

Comment:
Hm, normally physical devices of distinct instances should have disctinct handle (=pointer) value. Maybe this is asking for more problems?

Why not just `if` the destruction in `vkDestroyInstance`?

Reply:
The behavior is unchanged from the original design, a single PDev, which seems fine for how the mock ICD is used, though it could cause trouble in a pathological case.  I don't understand "'if' the destruction" -- the PDev should not be destroyed as long as there other instances exist, which'd mean tracking the instances, but maybe I misunderstood?

Reply:
Ah, right, it is `static` anyway.